assets/scripts/build_package_gallery.py

```diff
@@ -52,8 +54,9 @@ def gather_all_package_manifests(package_paths: List[str]) -> List[dict]:
     root = os.getcwd()
     for path in package_paths:
         try:
-            # Go to package, read manifest, and sync it
+            # Go to package, install deps, read manifest, and sync it
             os.chdir(path)
+            _pip_install("-r", "requirements.txt")
```

Comment:
I'm not sure how the pip package is meant to be used in code, but I got an error `no such option: -r` when running build_package_gallery.py

```
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.

Usage:
  build_package_gallery.py <command> [options]

no such option: -r
```